packages/schemix/src/schemix/__init__.py

```
@@ -1,2 +1,75 @@
+"""Schemix ORM - Type-safe ORM for Python."""
+
+# Core table and column types
+from schemix.columns import (
```

Pylint (3.3.7)

[error] Cannot import 'schemix.columns' due to 'invalid syntax (schemix.columns, line 36)'

(E0001)